Flutter/DateRangePicker/customizations.md

```diff
@@ -169,6 +172,9 @@ Widget build(BuildContext context) {
 
 ![Year cell customization Date Range Picker](images/customizations/yearcell_customization.png)
 
+>**NOTE**
+* Cell decoration will be rendered in this order disable dates, today date, trailing and leading dates and dates decoration based on the given decoration.
```

Comment:
Refer the above comment.

Reply:
I have changed and updated this in my latest commit